### PR TITLE
Commiting

### PR DESCRIPTION
**Pull request by kevinouyang1**
- Created for future python code reviews
- Testing code reviews and resolving issues on python files

### CODE REVIEW

File: python_code_review/test_review.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Test:
+	def __init__(self, message):
+		self.message = message
+
+	def set_message(self, text):
+		self.message = text
+
+
+print("hello world")
```
Comment (charlieouyang) [inline python_code_review/test_review.py:9]: This is a test message

File: python_code_review/test_review.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+computer = Test("Hello World")
+
+print(computer.message)
+
+computer.set_message("Leaving World")
+
+print(computer.message)
```
Comment (charlieouyang) [inline python_code_review/test_review.py:19]: This is another test message. Reply to this please

File: python_code_review/test_review.py (hunk 1 above)
Comment (kevinouyang1) [reply python_code_review/test_review.py:9]: Good

File: python_code_review/test_review.py (hunk 2 above)
Comment (kevinouyang1) [reply python_code_review/test_review.py:19]: Changed, please take another look

**Pull request merged**